File: fizzbuzz.cpp
#include <iostream>

int main(){ 
  for(int i = 1; i <101; i++){ //run a loop from 1 to 100
    if(i % 3 == 0 && i % 5 == 0){ //if i is divisible by 3 or 5
      std::cout << "fizzbuzz\n";  //output fizzbuzz
    }
    else if(i % 3 == 0){  //if i is dvisible by 3
      std::cout << "fizz\n";  //output fizz
    }
    else if(i % 5 == 0){  //if i is divisible by 5
      std::cout << "buzz\n";  //output buzz
    }
    else{
      std::cout << i << "\n"; //else output number and an extra line
    }
  }
  return 0 ;
}

File: fizzbuzz.java
public class fizzbuzz 
{
    public static void main(String[] args) 
    {
        fizzbuzzMethod(100);    //call the method
    }
 
    private static void fizzbuzzMethod(int num) 
    {
        for (int i = 1; i <= num; i++) 
        {
            if ((i % 3 == 0) && (i % 5 == 0)) //if the number is divisible by 3 and 5
                System.out.println("fizzbuzz");
            else if (i % 3 == 0) //if the number is divisible by 3
                System.out.println("fizz");
            else if (i % 5 == 0) //if the number is divisble by 5
                System.out.println("buzz");
            else
                System.out.println(i); //else output the original number
        }
    }
}